metisp/workflows/metis/metis_parameters.yaml

```diff
@@ -1,18 +1,21 @@
-default_parameters: &default
+default_parameters:
   is_default: yes
   recipe_parameters:
-    dark_task:
+    dark_task: &dark_task
       # Overriding the default for stacking
       metis.metis_det_dark.stacking.method: "median"
+  
+  workflow_parameters:
+    molecfit: "science"
 
 qc0_parameters:
-  <<: *default
+  <<: *dark_task
```

Comment:
Not sure this works as intended, because I would guess that expanded this looks like:
```
qc0_parameters:
  metis.metis_det_dark.stacking.method: "median"
```
which seems wrong, because the entire `workflow_parameters` or `recipe_parameters` part is gone. But maybe I'm reading this incorrectly.